src/Microsoft.Health.Dicom.Client/DicomWebClient.Workitem.cs

```diff
@@ -35,6 +35,24 @@ public async Task<DicomWebResponse> CancelWorkitemAsync(IEnumerable<DicomDataset
         return await PostRequest(uri, dicomDatasets, cancellationToken).ConfigureAwait(false);
     }
 
+    public async Task<DicomWebResponse<DicomDataset>> RetrieveWorkitemAsync(string workitemUid, string partitionName = default, CancellationToken cancellationToken = default)
+    {
+        var requestUri = GenerateWorkitemRetrieveRequestUri(workitemUid, partitionName);
+
+        using var request = new HttpRequestMessage(HttpMethod.Get, requestUri);
+
+        request.Headers.Accept.Add(DicomWebConstants.MediaTypeApplicationDicomJson);
+
+        var response = await HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
+
+        await EnsureSuccessStatusCodeAsync(response).ConfigureAwait(false);
+
+        var contentValueFactory = new Func<HttpContent, Task<DicomDataset>>(
+            content => Deserialize<DicomDataset>(content));
```

Comment:
We don't need a new `Deserialize` method here - we can use the existing `ValueFactory` method in the client base class.